pygeoapi/flask_app.py

```diff
@@ -131,6 +161,11 @@ def dataset(feature_collection, feature=None):
 @APP.route('/processes')
 @APP.route('/processes/<name>')
 def describe_processes(name=None):
+    """
+    OGC open api processes access point (experimental)
+
+    :returns: HTTP response
```

Comment:
all `:returns:` should describe the data returned (`dict` of ..., `list` of ...), etc.